packages/metadata-manager/index.html

```diff
@@ -5,6 +5,9 @@
     <meta charset="utf-8">
 
     <title>Metadata manager</title>
+    
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/es6-promise/4.1.1/es6-promise.min.js"></script>
```

Comment:
@fdlk suggested for #46 to link to polyfill.js. For consistency purposes I think it is a good idea to do that here as well (or use CDN for both)